src/components/profile/user/EditDetails.js

```diff
@@ -0,0 +1,104 @@
+import React, { Fragment, useState } from 'react';
+
+// Components
+import CustomButton from '../../../util/CustomButton';
+import EditDetailsDisplay from './EditDetailsDisplay';
+
+// MUI
+import withStyles from '@material-ui/core/styles/withStyles';
+import style from '../../../style/style';
+
+// Icons
+import EditIcon from '@material-ui/icons/Edit';
+
+// Redux
+import { connect } from 'react-redux';
+import { editUserDetailAction } from '../../../redux/actions/userActions';
+
+/** Control how the user edits their bio information
+ * @type {React.FunctionComponent}
+ * @param {object} props
+ * @param {object} props.classes
+ * @param {object} props.bio
+ * @param {string} props.handle
+ * @param {any} props.editUserDetailAction
+ */
+const EditDetails = ({ classes, bio, handle, editUserDetailAction }) => {
+  const [editorState, setEditorState] = useState({
+    aboutMe: '',
+    website: '',
+    location: '',
+  });
+  const [openState, setOpenState] = useState({
```

Comment:
the point of useState is it internalizes the state object. you can simplify this to 
const [open, setOpen] = useState(false);

Reply:
fixed

Reply:
This doesn't look fixed :/

Reply:
oh...nvm github UI  confused me.